template/map.jinja

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8 -*-
 # vim: ft=jinja
 
-{## Start with  defaults from defaults.sls ##}
+{## Start with defaults from defaults.yaml ##}
```

Comment:
Another possibility is to remove line 4 altogether, since this is a quite redundant comment ...